playbooks/roles/nginx/templates/edx/app/nginx/sites-available/cms.j2

```diff
@@ -42,6 +42,9 @@ error_page {{ k }} {{ v }};
   add_header Strict-Transport-Security "max-age=31536000; includeSubDomains";
```

Comment:
i believe we've added other headers in the past by adding new configuration to yml files. i feel like that'd be preferable, right? @manikarthikk and @kabirkhan , what do you guys think?

Reply:
Yes @sdolenc . Moreover, i tentatively think that  https://github.com/Microsoft/edx-platform/pull/77 will resolve this one.  i will discuss with @btelnes to understand further.

Reply:
i spoke to Brain and Mani directly. This change makes sense to me as is. In fact, it's a good candidate for the upstream. Brian may open a story to track that effort